docs/source/release_notes.rst

```diff
@@ -12,9 +12,10 @@ Future Release
     * Documentation Changes
         * Prevent logging on build (:pr:`1498`)
     * Testing Changes
+        * Test featuretools on pandas 1.3.0 release candidate and make fixes (:pr:`1492`)
```

Comment:
Probably should update this to indicate we are now at the point of implementing changes for pandas 1.3.0 compatibility instead of testing changes on the release candidate.